### ISSUE TITLE
Oddness previewing URLs

### ISSUE THREAD

**Issue by AlkalineJaunt** [mantisbt-plugins/BBCodePlus#26]
Over at the [FreeCAD tracker](https://freecadweb.org/tracker) we use your plugin.

I have noticed that the comment preview exhibits some strange behaviour. In particular it doesn't render URLs correctly:
```
[quote="[url=https://www.freecadweb.org/tracker/bug_report_page.php]tracker[/url]"]
here's a quote with a [url=https://www.freecadweb.org/tracker/bug_report_page.php]url[/url] in the attribution and in the body
[/quote]
And it doesn't preview properly
[quote=Eric the Viking]
  but this one does
[/quote]
[quote=Eric the Viking]
  a quote with a [url=https://www.freecadweb.org/tracker/bug_report_page.php]url[/url] in it doesn't work either
[/quote]

[list]
[*]lists preview somewhat differently too...
[*]...kind of with odd spacing
[/list]
Here's a plain url to check: [url=https://www.freecadweb.org/tracker/bug_report_page.php]url[/url]
And the preview fails
https://www.freecadweb.org/tracker/bug_report_page.php
Without the url bbcode, it still doesn't render usefully.
```
![untitled](https://user-images.githubusercontent.com/1776123/29399297-c023b2d0-837c-11e7-8926-55bd8f72fbea.jpg)

**Comment by bkraul** [comment]
@apelly, make sure MantisBT Formatting plugin is either disabled, or has the following options disabled:

-URL Processing.
-MantisBT internal links.

There is an incompatibility between the plugin and these two features. Please let me know if this resolves your issue so I can close it.

**Comment by AlkalineJaunt** [comment]
Sorry about the delay. We chose to disable
-URL Processing.
-MantisBT internal links.
in the MantisBT formatting plugin.

This is an acceptable solution, but I stumbled across an odd preview duplication when testing. To reproduce:
-Add a word or two of note
-Click preview icon
-Add two new lines after your word or two.
-All of the text in the note is duplicated.

![screenshot from 2017-08-30 10-58-01](https://user-images.githubusercontent.com/1776123/29847949-331050ba-8d72-11e7-87dd-a1475d01c939.png)
